cuebot/src/test/java/com/imageworks/spcue/test/util/FrameSetTests.java

```diff
@@ -8,37 +8,37 @@
 
 public class FrameSetTests {
     @Test
-    public void testMultipleSegments() {
+    public void shouldSplitListAndMaintainOrder() {
```

Comment:
@bcipriano just curious if this `shoud*` naming should be used going forward?

Reply:
It's not really something we've settled as a project.

I like the "should" naming personally. I find it makes it really easy to read and understand what the test is testing. I'm sure there are various effective options for unit test naming conventions, and most of our tests currently do not follow any of those. :) I think the new naming in this PR is a big improvement on what's there.

So, I'm not sure if it's something we really need to decide at the TSC level, but in the absence of a better convention I think it would be good to go with "should" naming for now.